src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -1075,9 +1075,14 @@ def load_arguments(self, _):
             c.argument('public_ip_address', help='Name of the public IP address when creating one (default) or referencing an existing one. Can also reference an existing public IP by ID or specify "" for None (\'""\' in Azure CLI using PowerShell or --% operator). For Azure CLI using powershell core edition 7.3.4, specify '' or "" (--public-ip-address '' or --public-ip-address "")')
             c.argument('public_ip_address_allocation', help=None, default=None, arg_type=get_enum_type(['dynamic', 'static']))
             c.argument('public_ip_address_dns_name', help='Globally unique DNS name for a newly created public IP.')
-            if self.supported_api_version(min_api='2017-08-01', resource_type=ResourceType.MGMT_NETWORK):
+
+            if self.cli_ctx.cloud.profile == 'latest':
+                c.argument('public_ip_sku', help='Public IP SKU. It is set to Standard by default. The public IP is supported to be created on edge zone only when it is \'Standard\'',
+                           default=None, arg_type=get_enum_type(['Basic', 'Standard']))
```

Comment:
Could we set the `default=` to `'Standard'` to keep consistent with other SKU related parameters and

Reply:
The difference of setting default value "Standard" in _params.py and custom.py would result in a difference of the default value of "--public-ip-address-allocation", which is "Static" and "dynamic".
This may need a confirmation from service team.

Hi @cburuzie may I ask when "--public-ip-sku" set to "Standard", does the "--public-ip-address-allocation" is set to "Static" by default instead of "dynamic"?